### ISSUE TITLE
Aarch64 missing main binary

### ISSUE THREAD

**Issue by fayalalebrun** [flathub/com.github.ahrm.sioyek#22]
When trying to run the flatpak, I get:
```
bwrap: execvp sioyek: No such file or directory
```

It seems as though /app/bin/sioyek is a symbolic link to /app/sioyek/sioyek but the latter does not exist.

**Comment by nbenitez** [comment]
I don't have access to a Aarch64 machine right now, can you do the following?
` flatpak run --command=sh com.github.ahrm.sioyek` and then `ls /app/sioyek/` and paste the output here..

**Comment by fayalalebrun** [comment]
```
[📦 com.github.ahrm.sioyek ~]$ ls /app/sioyek/
keys.config  keys_user.config  prefs.config  prefs_user.config	shaders  tutorial.pdf
```

Looks like the sioyek binary is missing, maybe some part of the build fails but the error is ignored?

**Comment by nbenitez** [comment]
Also paste `ls /app/bin`

**Comment by fayalalebrun** [comment]
```
[📦 com.github.ahrm.sioyek ~]$ ls /app/bin/
sioyek
[📦 com.github.ahrm.sioyek ~]$ file /app/bin/sioyek
/app/bin/sioyek: broken symbolic link to /app/sioyek/sioyek
```

**Comment by nbenitez** [comment]
Ok, let's launch a build to later check the logs

Done in a pull request [better](https://github.com/flathub/com.github.ahrm.sioyek/pull/23#issuecomment-2474423591).

**Comment by nbenitez** [comment]
@fayalalebrun I'm testing a new update, can you test if this works ok in aarch64, instructions below:

To test this build, install it from the testing repository:

`flatpak install --user https://dl.flathub.org/build-repo/145169/com.github.ahrm.sioyek.flatpakref`

**Comment by fayalalebrun** [comment]
I get `QEGLPlatformContext: Failed to create context: 3009` but I think this is no longer an issue specific to the flatpak.

Related:
https://github.com/ahrm/sioyek/issues/1184
https://github.com/ahrm/sioyek/issues/1101

**Comment by nbenitez** [comment]
Ok, but at least the binary is there and can be run, thanks for testing, so flatpak packaging will be fine for next release.

But as you mention Sioyek itself would need to make OpenGL support optional so it does not have problems in Aarch64 where GL versions are older. Thanks for the issue links.

**Comment by nbenitez** [comment]
Closing as fixed in last flathub release 2.0.0.fl1
